api/src/app.ts

```diff
@@ -52,6 +54,7 @@ export const build = async (
   });
   // NOTE: Awaited to ensure `.use` is registered on `fastify`
   await fastify.register(middie);
+  await fastify.register(fastifySentry, { dsn: SENTRY_DSN });
```

Comment:
```suggestion
  if (SENTRY_DSN !== "dsn_from_sentry_dashboard") {
    await fastify.register(fastifySentry, { dsn: SENTRY_DSN });
  }
```

I assume we want similar logic as we have in the current API?

That is, don't initialise Sentry if a DSN wasn't provided?